Clarify difference between config-based and component routing (#1079)
solidjs/solid-docs: src/routes/solid-router/getting-started/config.mdx

title: "Config-based routing"
---

> Solid Router supports config-based routing, offering the same capabilities as [component routing](/solid-router/getting-started/component).

Reviewer: ```suggestion
Solid Router supports config-based routing, which offers the same capabilities as [component routing](/solid-router/getting-started/component).
```

> The choice of config-based routing over component routing is largely a matter of personal preference.

Reviewer: ```suggestion
The decision to use config-based routing over component routing depends largely on personal preference.
```
